compilers/concrete-compiler/compiler/tests/end_to_end_tests/end_to_end_jit_test.cc

```diff
@@ -460,3 +460,18 @@ func.func @main(%arg0: !FHE.eint<3>) -> !FHE.eint<3> {
 )XXX");
   assert(err.has_value());
 }
+
+TEST(CompileMultiOutputDoubleReturn, double_return) {
```

Comment:
should we also have some CP tests with alias-outputs being returned?

Reply:
I would say it is enough to have one in the compiler.